generate

```diff
@@ -158,7 +158,7 @@ def recurse(aspect, level=0):
                 output += "* `" + subsub_name + " <" + subsub_name + "/README.rst>`_\n"
         else:
             output += "This aspect does not have any sub aspects.\n\n"
-
+        output += "\n"
```

Comment:
Nope, I think this line should be inside the `if` (line 156). Because if the `else` branch you'd have three newlines but if the `if` branch executes, you'd have only one.